contracts/exchange-forwarder/test/forwarder.ts

```diff
@@ -6,61 +6,52 @@ import {
     chaiSetup,
     constants,
     ContractName,
-    ERC20BalancesByOwner,
-    expectContractCreationFailedAsync,
-    expectTransactionFailedAsync,
+    getLatestBlockTimestampAsync,
     OrderFactory,
     provider,
     sendTransactionResult,
     txDefaults,
     web3Wrapper,
 } from '@0x/contracts-test-utils';
 import { BlockchainLifecycle } from '@0x/dev-utils';
-import { assetDataUtils } from '@0x/order-utils';
-import { RevertReason, SignedOrder } from '@0x/types';
+import { assetDataUtils, ForwarderRevertErrors } from '@0x/order-utils';
 import { BigNumber, providerUtils } from '@0x/utils';
 import { Web3Wrapper } from '@0x/web3-wrapper';
 import * as chai from 'chai';
 import { TransactionReceiptWithDecodedLogs } from 'ethereum-types';
 
-import { artifacts, ForwarderContract, ForwarderWrapper } from '../src';
+import { artifacts, ForwarderContract, ForwarderTestFactory, ForwarderWrapper } from '../src';
 
 chaiSetup.configure();
 const expect = chai.expect;
 const blockchainLifecycle = new BlockchainLifecycle(web3Wrapper);
 const DECIMALS_DEFAULT = 18;
-const MAX_WETH_FILL_PERCENTAGE = 95;
 
 describe(ContractName.Forwarder, () => {
```

Comment:
We should probably upgrade these fixtures to use the new mocha extensions added in #2007 while we're here.